packages/react-app/src/components/pages/Bounties/Filters/index.tsx

```diff
@@ -258,15 +270,37 @@ const MinMaxFilter = ({ name, filters, setFilters }: {
 
 	return (
 		<>
-			{name && <Heading size="xs">{name}</Heading>}
 			<HStack my="2">
-				<Input placeholder="Min" onChange={updateMin} />
-				<Input placeholder="Max" onChange={updateMax} />
+				<Input placeholder="Min Bounty Value" onChange={updateMin} />
+				<Input placeholder="Max Bounty Value" onChange={updateMax} />
 			</HStack>
 		</>
 	);
 };
 
+const SaveFilter = ({ filters }: {filters: FilterParams}): JSX.Element => {
+	const { colorMode } = useColorMode();
+	const [showModal, setShowModal] = useState(false);
+	const { user } = useUser();
+	
+	const handleSaveFilter = () => {
+		setShowModal(true);
+	};
+
+	if (!user) return <></>;
```

Comment:
I like this - though I do wonder since the spacing shifts a fair amount from the unlogged in to the logged in state for the user if we could better minimize that

Reply:
As per the discussion, we will be taking up the design changes as a separate CR. This one is more about adding functionality